### PR TITLE
Add 2020 Week 24 release announcements

### PR DESCRIPTION
**Pull request by cokiengchiara**
Towards https://github.com/giantswarm/giantswarm/issues/11416

- [x] Waiting for teams' contributions for this week's releases. Will announce on Friday 2pm CET.

Listed in decreasing order of (subjective) relevance.

*Questions*
- [x] Solved. Links look all over the place in Slack. How to write in markdown so that links show as hyperlinks in Slack? This doesn’t help https://slack.com/intl/en-de/help/articles/202288908-Format-your-messages
- [x] Solved. What do you think of the date format? I think it makes sense for the date to be Friday of that week, since that is when the announcement is being made.

https://share.getcloudapp.com/xQuDZdXm links look super messy

### CODE REVIEW

**Comment by marians** [comment]
Can we please have a `v` prefix on every version number?

File: announcements/2020-Week-24.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# This Week in Releases (12.06.2020)
```
Comment (marians) [inline announcements/2020-Week-24.md:1]:
```suggestion
# This Week in Releases (June 12, 2020)
```

DD.MM.YYYY is not a date format that makes sense to everybody internationally. Spelling out the month is really the only way you can make it pretty much fail safe.

File: announcements/2020-Week-24.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+1-**CoreOS is replaced with Flatcar Container Linux** in [AWS 9.0.6](https://github.com/giantswarm/releases/blob/master/aws/v9.0.6/release-notes.md)
```
Comment (marians) [inline announcements/2020-Week-24.md:3]: I would prefer a `1. ` (with a blank) over the `1-` without blank.

File: announcements/2020-Week-24.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# This Week in Releases (June 12, 2020)
+
+1-**CoreOS is replaced with Flatcar Container Linux** in [AWS 9.0.6](https://github.com/giantswarm/releases/blob/master/aws/v9.0.6/release-notes.md)
```
Comment (cokiengchiara) [inline announcements/2020-Week-24.md:3]:
```suggestion
1. **CoreOS is replaced with Flatcar Container Linux** in [AWS 9.0.6](https://github.com/giantswarm/releases/blob/master/aws/v9.0.6/release-notes.md)
```

File: announcements/2020-Week-24.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
+# This Week in Releases (June 12, 2020)
+
+1. **CoreOS is replaced with Flatcar Container Linux** in [AWS 9.0.6](https://github.com/giantswarm/releases/blob/master/aws/v9.0.6/release-notes.md)
+
+2-Reliability of NGINX Ingress Controller is improved to **speed up recovery when NGINX gets overloaded** in [AWS 9.3.3](https://github.com/giantswarm/releases/blob/master/aws/v9.3.3/release-notes.md), [AWS 9.0.6](https://github.com/giantswarm/releases/blob/master/aws/v9.0.6/release-notes.md), and [NGINX IC 1.6.12](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#v1612-2020-06-04) (app can be installed from the Giant Swarm App Catalog)
```
Comment (cokiengchiara) [inline announcements/2020-Week-24.md:5]:
```suggestion
2. Reliability of NGINX Ingress Controller is improved to **speed up recovery when NGINX gets overloaded** in [AWS 9.3.3](https://github.com/giantswarm/releases/blob/master/aws/v9.3.3/release-notes.md), [AWS 9.0.6](https://github.com/giantswarm/releases/blob/master/aws/v9.0.6/release-notes.md), and [NGINX IC 1.6.12](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#v1612-2020-06-04) (app can be installed from the Giant Swarm App Catalog)
```

File: announcements/2020-Week-24.md (hunk 2 above)
Comment (cokiengchiara) [reply announcements/2020-Week-24.md]: Thanks! Done

**Comment by cokiengchiara** [comment]
- [ ] Create Release Notes Style Guide.md

Release notes style guide
1. Use date format like "June 12, 2020" to work internationally and avoid ambiguity
2. Every version number should have `v`, e.g. `v1.1.1`
3. Use `<https://link.com/|My Text>` to show links as hyperlinks (and not show the whole URL) in Slack. However, it only works when using the API to post (not when manually typed).

File: announcements/2020-Week-24.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+5. [**Kubernetes GPU**](https://github.com/giantswarm/kubernetes-gpu/blob/master/README.md) can now be installed as an app from the Playground Catalog. For users that want to run GPU workloads, this app makes sure required special drivers (CUDA) are present and installed in the right version in the GPU machines.
+
+
+---
+
+Slack Markdown
+Note: Use `<https://link.com/|My Text>` to show links as hyperlinks (and not show the whole URL) in Slack. However, it only works when using the API to post (not when manually typed).
+
+```
+:zap: *This Week in Releases (12.06.2020)* :zap:
```
Comment (cokiengchiara) [inline announcements/2020-Week-24.md:20]:
```suggestion
:zap: *This Week in Releases (June 12, 2020)* :zap: 
```

File: announcements/2020-Week-24.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+
+3. *An issue causing cert-manager to be killed* for exceeding its memory limit, stopping SSL certificates from being automatically renewed *is fixed* in <https://github.com/giantswarm/releases/blob/master/aws/v11.3.3/release-notes.md|AWS v11.3.3>
+
+4. *Kong is upgraded* to the latest upstream Kong v1.6.1 and Kong Ingress Controller v0.9.0 in <https://github.com/giantswarm/kong-app/blob/master/CHANGELOG.md#v081---2020-06-02|Kong v0.8.1> (app can be installed from the Giant Swarm App Catalog)
+
+5. <https://github.com/giantswarm/kubernetes-gpu/blob/master/README.md|*Kubernetes GPU*> can now be installed as an app from the Playground Catalog. For users that want to run GPU workloads, this app makes sure required special drivers (CUDA) are present and installed in the right version in the GPU machines.
+```
```
Comment (cokiengchiara) [inline announcements/2020-Week-24.md:31]:
```suggestion
---

Please help us improve our communication. *Was this helpful?* :thumbsup::skin-tone-3: :thumbsdown::skin-tone-3:
```

File: announcements/2020-Week-24.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+
+3. *An issue causing cert-manager to be killed* for exceeding its memory limit, stopping SSL certificates from being automatically renewed *is fixed* in <https://github.com/giantswarm/releases/blob/master/aws/v11.3.3/release-notes.md|AWS v11.3.3>
+
+4. *Kong is upgraded* to the latest upstream Kong v1.6.1 and Kong Ingress Controller v0.9.0 in <https://github.com/giantswarm/kong-app/blob/master/CHANGELOG.md#v081---2020-06-02|Kong v0.8.1> (app can be installed from the Giant Swarm App Catalog)
+
+5. <https://github.com/giantswarm/kubernetes-gpu/blob/master/README.md|*Kubernetes GPU*> can now be installed as an app from the Playground Catalog. For users that want to run GPU workloads, this app makes sure required special drivers (CUDA) are present and installed in the right version in the GPU machines.
+---
+
+Please help us improve our communication. *Was this helpful?* :thumbsup::skin-tone-3: :thumbsdown::skin-tone-3:
```
Comment (cokiengchiara) [inline announcements/2020-Week-24.md:33]:
```suggestion
Please help us improve our communication. *Was this helpful?* :thumbsup::skin-tone-3: :thumbsdown::skin-tone-3:
```
```

Comment (cokiengchiara) [inline announcements/2020-Week-24.md:33]:
```suggestion
Please help us improve our communication. *Was this helpful?* :thumbsup::skin-tone-3: :thumbsdown::skin-tone-3:
```
```

**Comment by cokiengchiara** [comment]
Done and announced.

**Pull request merged**